detection_rules/integrations.py

```diff
@@ -47,12 +47,13 @@ class IntegrationManifestSchema(Schema):
     description = fields.Str(required=True)
     download = fields.Str(required=True)
     conditions = fields.Dict(required=True)
-    policy_templates = fields.List(fields.Dict, required=True)
+    policy_templates = fields.List(fields.Dict)
```

Comment:
Does this mean not all integrations have a policy_template? Should they?

Reply:
Sorry for the noob q, but what does a policy template mean?

Reply:
Confirming that not all integrations have policy templates, so this is a valid change.

Reply:
@Mikaayenson policy templates are not a requirement for packages, so when we originally made this dataclass we may have added more constraints than required. @ajosh0504 policy templates in the manifest are how Fleet packages distinguish between a package (AWS) and it's actual integrations (CloudTrail). So for example, in the AWS package, CloudTrail would have its own section (policy template) in the AWS manifest. 

Endpoint is a similar package/integration, but the package `endpoint` has the `endpoint` policy template in it. The packages are still valid without a policy template so no need to adjust the analytic packages manifest.